Google.PowerShell/ReleaseFiles/GoogleCloudPlatform.Format.ps1xml

```diff
@@ -162,5 +162,52 @@
         </TableRowEntries>
       </TableControl>
     </View>
+    <View>
+      <Name>BigQuery Data Row</Name>
```

Comment:
I'm referring to this [TableView](https://msdn.microsoft.com/en-us/library/dd901841(v=vs.85).aspx)